evaluate/report/csv_test.go

```diff
@@ -32,57 +33,63 @@ func TestGenerateCSVForAssessmentPerModelPerLanguagePerRepository(t *testing.T)
 	validate(t, &testCase{
 		Name: "Single Empty Model",
 
-		Assessments: AssessmentPerModelPerLanguagePerRepository{
+		Assessments: AssessmentPerModelPerLanguagePerRepositoryPerTask{
 			modeltesting.NewMockModelNamed(t, "some-model"): {
 				languagetesting.NewMockLanguageNamed(t, "some-language"): {
-					"some-repository": metrics.NewAssessments(),
+					"some-repository": {
+						task.IdentifierWriteTests: metrics.NewAssessments(),
+					},
 				},
 			},
 		},
 
 		ExpectedString: `
 			model,language,repository,score,coverage,files-executed,generate-tests-for-file-character-count,processing-time,response-character-count,response-no-error,response-no-excess,response-with-code
-			some-model,some-language,some-repository,0,0,0,0,0,0,0,0,0
+			some-model,some-language,some-repository,write-tests,0,0,0,0,0,0,0,0,0
 		`,
 	})
 	validate(t, &testCase{
 		Name: "Multiple Models",
 
-		Assessments: AssessmentPerModelPerLanguagePerRepository{
+		Assessments: AssessmentPerModelPerLanguagePerRepositoryPerTask{
 			modeltesting.NewMockModelNamed(t, "some-model-a"): {
 				languagetesting.NewMockLanguageNamed(t, "some-language"): {
-					"some-repository": metrics.Assessments{
-						metrics.AssessmentKeyGenerateTestsForFileCharacterCount: 50,
-						metrics.AssessmentKeyResponseCharacterCount:             100,
-						metrics.AssessmentKeyCoverage:                           1,
-						metrics.AssessmentKeyFilesExecuted:                      2,
-						metrics.AssessmentKeyResponseNoError:                    3,
-						metrics.AssessmentKeyResponseNoExcess:                   4,
-						metrics.AssessmentKeyResponseWithCode:                   5,
-						metrics.AssessmentKeyProcessingTime:                     200,
+					"some-repository": {
```

Comment:
Please use the type for composite literals.

Reply:
The tests were already like this. And since I'm planning on changing this with #169 I'm not really eager to change this now.